kdc/kx509.c

```diff
@@ -770,11 +730,18 @@ check_authz(krb5_context context,
     if (KeyUsage2int(ku) != (KeyUsage2int(ku) & KeyUsage2int(ku_allowed)))
         goto eacces;
 
+    _kdc_audit_addkv((kdc_request_t)reqctx, 0, "authorized", "true");
     return 0;
 
 eacces:
     ret = EACCES;
+    goto out2;
+
 out:
+    /* XXX Display error code */
+    _kdc_audit_addreason((kdc_request_t)reqctx,
+                         "Error handling requested extensions");
```

Comment:
If "out fault", a level-1 log message may be appropriate...

Reply:
Can't quite tell here.  The utility functions doing the work should call `kdc_log()` themselves.